A32NX/html_ui/Pages/A32NX_Core/A32NX_APU.js

```diff
@@ -6,12 +6,31 @@ class A32NX_APU {
         console.log('A32NX_APU init');
         SimVar.SetSimVarValue("L:APU_FLAP_OPEN", "Percent", 0);
         this.lastAPUBleedState = -1;
+
+        // FIX FOR 1.8.3 UPDATE
+        SimVar.SetSimVarValue("L:A32NX_FUEL_FIX_APPLIED", "Enum", -1);
```

Comment:
I am not familiar with the SimVar, so please ignore if it's not a valid question. Why not use boolean instead of Enum type here?

Reply:
The fix simvar is a 3-state value.

Reply:
i agree boolean would be better, but its a temp fix and will be removed soon hopefully